### DISCUSSION TITLE
Error in TSSEnrichment

### DISCUSSION THREAD

**Discussion by singcell** [stuart-lab/signac#996] [General]
When I run `TSSEnrichment` for rat genome.

pbmc <- TSSEnrichment(object = pbmc, fast = TRUE)
Extracting TSS positions
Extracting fragments at TSSs
The index file is older than the data file:

|--------------------------------------------------|
|==================================================|

I get this error message

"Error in width(x = region)[[1]] : subscript out of bounds"

**Comment by timoast** [comment]
Please see the past issues related to `TSSEnrichment()` by searching the github issues page. If the suggestions in those issues does not solve your problem, please open a new issue containing the full code, the output of `sessionInfo()`, the first few lines of the gene annotations you're using, and the first few lines of the fragment file you're using.
